magik-checks/src/main/java/nl/ramsolutions/sw/checks/magik/UseValueCompareCheck.java

```diff
@@ -14,15 +14,14 @@ public class UseValueCompareCheck extends MagikCheck {
   @SuppressWarnings("checkstyle:JavadocVariable")
   public static final String CHECK_KEY = "UseValueCompare";
 
-  private static final String MESSAGE = "Type '%s' should not be compare with _is.";
+  private static final String MESSAGE = "Type should not be compare with _is/_isnt.";
```

Comment:
```suggestion
  private static final String MESSAGE = "Lhs and rhs should not be compare with _is/_isnt.";
```

`Type` sounds a bit odd in this context at the moment, doesn't it?

Reply:
Or maybe even:

```suggestion
  private static final String MESSAGE = "Strings, bignums and floats should not be compare with _is/_isnt.";
```